src/WCSimConstructPMT.cc

```diff
@@ -379,63 +379,25 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
     ////////////////////
     /// 1-PMT support //
     ////////////////////
-    //54mm is position of full PMT support wrt inside of pressure vessel.
-    G4Cons * solidWCPMTsupport =
-      new G4Cons("WCPMTsupport",
-		 0.,                                                                      //rmin1
-		 tan(mPMT_pmt_openingAngle)*(vessel_radius_curv - mPMT_outer_material_d - pmtModuleHeight),      //rmax1
-		 0.,                                                                      //rmin2
-		 tan(mPMT_pmt_openingAngle)*(vessel_radius_curv - mPMT_outer_material_d
-			       -expose - dist_pmt_vessel),                                //rmax2
-		 (pmtModuleHeight - expose - dist_pmt_vessel)/2,                                   //h/2
-		 0.0*deg,                                                                 //phiStart
-		 360.0*deg);                                                              //Deltaphi
-
-    G4LogicalVolume* logicWCPMTsupport =
-      new G4LogicalVolume(solidWCPMTsupport,
-			  G4Material::GetMaterial("Blacksheet"), 
-			  "WCPMTsupport",
-			  0,0,0);
-
-    new G4LogicalSkinSurface("FoamLogSkinSurface",logicWCPMTsupport,OpGelFoamSurface);
-
-    new G4PVPlacement(0,
-		      G4ThreeVector(0, 0, wcpmt_z_offset+(pmtModuleHeight - expose - dist_pmt_vessel)/2),
-		      logicWCPMTsupport,
-		      "WCPMTsupport",
-		      logicWCPMT,
-		      false,
-		      0,
-		      checkOverlaps);
 
     // Visualize
     G4VisAttributes* WCPMTVisAtt_sup = new G4VisAttributes(G4Colour(0.3,0.3,0.3));
     WCPMTVisAtt_sup->SetForceSolid(true);
-    logicWCPMTsupport->SetVisAttributes(WCPMTVisAtt_sup);
-  
 
     //Reflector support
     if(id_reflector_height > 0.1*CLHEP::mm 
        && (reflectorRadius-radius) > -5*CLHEP::mm){
       
-      G4double ReflectorHolderZ[3] = {0, id_reflector_z_offset, id_reflector_z_offset+id_reflector_height};
-      G4double ReflectorHolderR[3] = {tan(mPMT_pmt_openingAngle)*(vessel_radius_curv - mPMT_outer_material_d
-						    -expose - dist_pmt_vessel),
-				      tan(mPMT_pmt_openingAngle)*(vessel_radius_curv - mPMT_outer_material_d
-						    -expose - dist_pmt_vessel+id_reflector_z_offset),
-				      tan(mPMT_pmt_openingAngle)*(vessel_radius_curv - mPMT_outer_material_d
-						    -expose - dist_pmt_vessel+id_reflector_z_offset
-						    +id_reflector_height)};
+      G4double ReflectorHolderZ[4] = {0, 22.89, 42.04, 47.18};
+      G4double ReflectorHolderR[4] = {39.735, 43.15, 45.89, 46.615};
```

Comment:
Should these be assigned to variables instead? i.e they no longer correspond to `id_reflector_z_offset`, `id_reflector_height`, `vessel_radius_curv`, etc.?

Reply:
So, the above two lines actually represent the polycone shape which I used to define the support structure for the entire PMT where the z values represent the plane, r represents the inner radius and R represents the outer radius. These do not correspond to the variables you have mentioned above. They are different. `id_reflector_z_offset` gives the offset values with respect to the PMT placement. `id_reflector_height` is defined in WCSimDetectorConfigs.cc which is basically the vertical height of the reflector.`vessel_radius_curv` is the radius of curvature of the dome.